benchmarks/ops/benchmark_delta_rule.py

```
@@ -2,12 +2,12 @@
 # pip install "git+https://github.com/openai/triton.git#egg=triton&subdirectory=python"
 
 import torch
-from benchmark import benchmark_combined, benchmark_forward, benchmark_backward
+from benchmark import benchmark_backward, benchmark_combined, benchmark_forward
 
-from fla.ops.delta_rule import (chunk_delta_rule,
-                                fused_recurrent_delta_rule, fused_chunk_delta_rule)
+from fla.ops.delta_rule import chunk_delta_rule, fused_chunk_delta_rule, fused_recurrent_delta_rule
```

Ruff (0.8.2)

`fla.ops.delta_rule.fused_recurrent_delta_rule` imported but unused

Remove unused import: `fla.ops.delta_rule.fused_recurrent_delta_rule`

(F401)